### ISSUE TITLE
For version 1.7.1 all Controls to script disappeared at some point during the last week of June; for both Chrome and Firefox; Windows and Ubuntu Linux [BUG]

### ISSUE THREAD

**Issue by rangervaloy** [avallete/yt-playlists-delete-enhancer#259]
**Steps to reproduce**:

**Expected results**:

**Actual results**:

**Notes**:

**Comment by rangervaloy** [comment]
Resetting Chrome and installing a fresh copy of Firefox in Windows system didn't solve the issue! Is anyone able to run it?

**Comment by GaabluhnMowd** [comment]
Windows 10LTSC, Firefox (latest), can confirm script now seems to launch/activate only on the Watch Later playlist, but no custom playlists.

**Comment by rangervaloy** [comment]
Something must have changed in Youtube that made the script fail, because this and previous versions were working at some point, and now don't.

**Comment by GaabluhnMowd** [comment]
But still prevalent in 1.7.2

**Comment by GaabluhnMowd** [comment]
Did some more testing - create a new email and YouTube account, and now the script works fine, so in *my* particular case it's account related, and no doubt something to do with the fact I'm using a brand account for playlists - as when I tested the original accounts, I noticed the script would start, but as they aren't the "owner" of the playlist (the brand account is), the functions were disabled.

**Comment by rangervaloy** [comment]
I was thinking of resetting Chrome to see if the issue would go away, but after reading your last comment (GaabluhnMowd) I can confirm the script works on my wife's account on another Chrome tab! The same thing happens on Firefox: it doesn't work for my account, but it does for my wife's, on a different tab of the same window! It happens on both Linux and Windows. I disabled all extensions (except Tampermonkey); no luck.

**Comment by GaabluhnMowd** [comment]
Yes, with me, it's specific to brand accounts, because if I use any of the actual "owner" accounts tied to the brand, it loads up fine.  Which is a pain, as I use brand accounts so I can use my YouTube profile on various devices without having to switch google accounts. Oh well.

Hope you have better luck sorting it out.
